response_operations_ui/common/date_restriction_generator.py

```diff
@@ -1,5 +1,6 @@
 def get_date_restriction_text(tag, events):
-    return {
+
```

Comment:
I know you didn't write the initial code, but could you add a docstring to explain what this function does?  It's a bit tough to follow